workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/SelectPaymentMethod/components/BillingAddress/BillingAddress.tsx

```diff
@@ -0,0 +1,244 @@
+					<div className="align-items-center billing-address-section-card-header d-flex justify-content-between w-100">
+						<span className="billing-address-section-card-header-left-content">
+							New Address
+						</span>
+
+						<button
+							className="px-4 py-2"
+							onClick={() => {
+								setShowNewAddressButton(true);
+								setSelectedAddress('');
+
+								const billingAddress: BillingAddress = {
+									city: '',
+									country: '',
+									countryISOCode: 'US',
+									name: '',
+									phoneNumber: '',
+									regionISOCode: '',
+									street1: '',
+									street2: '',
+									zip: '',
+								};
+
+								setBillingAddress(billingAddress);
+							}}
+						>
+							Cancel
+						</button>
+					</div>
+
+					<div className="billing-address-section-container d-flex flex-column p-4 w-100">
+						<Input
+							label="Full Name"
+							onChange={({target}) => {
+								setBillingAddress({
+									...billingAddress,
+									name: target.value,
+								});
+							}}
+							required
+							value={billingAddress?.name}
+						/>
+
+						<Input
+							label="Address"
+							onChange={({target}) => {
+								setBillingAddress({
+									...billingAddress,
+									street1: target.value,
+								});
+							}}
+							required
+							value={billingAddress?.street1}
+						/>
+
+						<Input
+							onChange={({target}) => {
+								setBillingAddress({
+									...billingAddress,
+									street2: target.value,
+								});
+							}}
+							value={billingAddress?.street2}
+						/>
+
+						<div className="billing-address-double-input">
+							<Input
+								label="City"
+								onChange={({target}) => {
+									setBillingAddress({
+										...billingAddress,
+										city: target.value,
+									});
+								}}
+								required
+								value={billingAddress?.city}
+							/>
+
+							<Input
+								label="State"
+								onChange={({target}) => {
+									setBillingAddress({
+										...billingAddress,
+										regionISOCode: target.value,
+									});
+								}}
+								required
+								value={billingAddress?.regionISOCode}
+							/>
+						</div>
+
+						<div className="billing-address-double-input">
+							<Input
+								label="Zip/Area Code"
+								onChange={({target}) => {
+									setBillingAddress({
+										...billingAddress,
+										zip: target.value,
+									});
+								}}
+								required
+								value={billingAddress?.zip}
+							/>
+
+							<Input
+								label="Country"
+								onChange={({target}) => {
+									setBillingAddress({
+										...billingAddress,
+										country: target.value,
+									});
+								}}
+								required
+								value={billingAddress?.country}
+							/>
+						</div>
+
+						<Input
+							label="Phone"
+							onChange={({target}) => {
+								setBillingAddress({
+									...billingAddress,
+									phoneNumber: target.value,
+								});
+							}}
+							required
```

Comment:
Please, remove the {} here.
```suggestion
							onChange={({target}) => 
								setBillingAddress({
									...billingAddress,
									phoneNumber: target.value,
								});
							}
							required
```